server/routes/api/ringdowns.js

```diff
@@ -195,13 +195,17 @@ router.patch('/:id/deliveryStatus', middleware.isAuthenticated, async (req, res)
         req.body.deliveryStatus === DeliveryStatus.REDIRECT_ACKNOWLEDGED
       ) {
         // check if user is in the receiving hospital ED
-        const hospitalUser = await models.HospitalUser.findOne({
+        const options = {
           where: {
             HospitalId: patientDelivery.HospitalId,
             EdAdminUserId: req.user.id,
           },
           transaction,
-        });
+        };
+        if (process.env.REACT_APP_PILOT_SHOW_ALL_RINGDOWNS === 'true') {
+          delete options.where.HospitalId;
+        }
+        const hospitalUser = await models.HospitalUser.findOne(options);
```

Comment:
This allows any hospital ed user to update the ringdown status of any ringdown regardless of hospital destination.